Makefile

```diff
@@ -43,7 +43,8 @@ lint:
 .PHONY: full-test
 full-test:
 	@echo "Running all tests"
-	@go test -tags blst_enabled -timeout 20m ${COV_CMD} -race -p 1 -v ./...
+	@go test -tags blst_enabled -timeout 20m ${COV_CMD} -race -p 1 -v $(shell go list ./... | grep -v /integration/)
```

Comment:
You can remove this change and pull from stage where I fixed this in https://github.com/bloxapp/ssv/commit/6b4cde4cc1774d3b07e605d1782bd75b4e0848e4